src/pythium/sklimming/reader.py

```diff
@@ -22,14 +22,34 @@
 from pythium.sklimming import writer
 from pythium.common import tools
 
-
+try:
+    import XRootD.client as client
+    import XRootD.client.glob_funcs as remote_glob
+    remote_glob_imported = True
+except ImportError:
+    remote_glob_imported = False
```

Comment:
this is good, but I think you can save yourself some effort and boilerplate f you jsut do 

```
REMOTE_GLOBE_IMPORTED = False
try:
     import XRootD.client as client
     import XRootD.client.glob_funcs as glob
     REMOTE_GLOBE_IMPORTED = True
except ImportError:
      from glob import glob
```
(note also all caps for global)
then later on you just need to use glob once when looking for files